arrays_strings/fizz_buzz/fizz_buzz_solution.ipynb

```diff
@@ -117,17 +117,7 @@
     "            raise TypeError('num cannot be None')\n",
     "        if num < 1:\n",
     "            raise ValueError('num cannot be less than one')\n",
-    "        results = []\n",
-    "        for i in range(1, num + 1):\n",
-    "            if i % 3 == 0 and i % 5 == 0:\n",
-    "                results.append('FizzBuzz')\n",
-    "            elif i % 3 == 0:\n",
-    "                results.append('Fizz')\n",
-    "            elif i % 5 == 0:\n",
-    "                results.append('Buzz')\n",
-    "            else:\n",
-    "                results.append(str(i))\n",
-    "        return results"
+    " 	     return ['Fizz' * (i % 3 == 0) + 'Buzz' * (i % 5 == 0) or str(i) for i in range (1, num + 1)]"
```

Comment:
Oh God!. Good line :+1: 

But I think that the purpose of this repo is solve algorithms without the advice of the language.

I don't know what is the @donnemartin 's opinion.